test/test_pipeline/test_partialcredit.py

```diff
@@ -27,3 +27,19 @@ def test_no_credit(self):
         self.assertIn("ERROR:root:['ls'] should have exited unsuccessfully.", logs.output)
         self.assertEqual(results.score, 0)
         return
+
+    def test_list_credit(self):
+        pipelines = map(lambda t: Pipeline(Run(['ls'])), range(2))
+        values = [10, 1]
+        with self.assertLogs() as logs:
+            results = PartialCredit(pipelines, values)()
+        self.assertEqual(results.score, 11)
+        return
```

Comment:
This is a fantastic test. Very clear, very readable. Should we do anything with logs? And, what are we expecting to be logged every time a PartialCredit runs?

Reply:
If we don't want to do anything with logs, perhaps dropping the with guard here would be good

Reply:
Yeah, that was just a leftover from copy-pasting the style of the other tests. We don't actually need the logs here. I'll fix it in a bit and commit that

Reply:
Once that's in I think we can get this merged up.

Reply:
Okay, I pushed the changes! This should be all finished. Let me know if there's anything else!